CHANGELOG.rst

```diff
@@ -11,9 +11,12 @@ Change Log
 
 .. There should always be an "Unreleased" section for changes pending release.
 
-Unreleased
-~~~~~~~~~~
+[1.0.0] - 2020-10-13
+~~~~~~~~~~~~~~~~~~~~
 
+* Fix ``SettingDictToggle`` constructor.
```

Comment:
Is this fixing a consumer-visible problem? If so, we should mention which one in the changelog.  If not, it doesn't need to be in the changelog.

Reply:
@nedbat: FYI: @regisb may clarify this, but he's also working on the first PR to ever use this class, so it really won't affect anyone but him.

Reply:
Sounds like we don't need it in the changelog.

Reply:
I think we do need it in the changelog, as edx-toggles==0.3.0 was already released on pypi. I'll improve the phrasing.